src/PeerDigest.h

```diff
@@ -76,14 +76,14 @@ class PeerDigest
     CBDATA_CLASS(PeerDigest);
 
 public:
-    PeerDigest(CachePeer *);
+    PeerDigest(const CachePeer &);
```

Comment:
If we can ban implicit conversions, let's do that in this PR:

```suggestion
    explicit PeerDigest(const CachePeer &);
```

Reply:
Done.